<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1607</width>
    <height>562</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1607</width>
    <height>562</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1607</width>
    <height>562</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>501</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Display</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(71, 71, 71)</string>
    </property>
    <property name="text">
     <string>Clothing size Predictor</string>
    </property>
   </widget>
   <widget class="Line" name="main_line">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>160</y>
      <width>1771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(227, 227, 227);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>17</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="predictor" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>179</y>
      <width>1591</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Display</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(71, 71, 71)</string>
     </property>
     <property name="text">
      <string>Weight (kg)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>50</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Display</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(71, 71, 71)</string>
     </property>
     <property name="text">
      <string>Age (yrs)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>1205</x>
       <y>50</y>
       <width>160</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Display</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(71, 71, 71)</string>
     </property>
     <property name="text">
      <string>Height (cm)</string>
     </property>
    </widget>
    <widget class="QLabel" name="result">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>210</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Display</family>
       <pointsize>33</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(71, 71, 71)</string>
     </property>
     <property name="text">
      <string>Your clothing size is:</string>
     </property>
    </widget>
    <widget class="QLabel" name="prediction">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>212</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Display</family>
       <pointsize>31</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(103, 203, 116);</string>
     </property>
     <property name="text">
      <string>Small</string>
     </property>
    </widget>
    <widget class="QPushButton" name="predict_bt">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>290</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Display</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color: rgb(71, 71, 71);

color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Predict</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="weight_value">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>105</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF UI Display</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string comment="hola" extracomment="hola"/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="age_value">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>105</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF UI Display</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string comment="hola" extracomment="hola"/>
     </property>
     <property name="maximum">
      <number>117</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="height_value">
     <property name="geometry">
      <rect>
       <x>1200</x>
       <y>105</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF UI Display</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string comment="hola" extracomment="hola"/>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QLabel" name="valid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 98, 98)</string>
     </property>
     <property name="text">
      <string>Please introduce valid values</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="predictor_line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>801</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(97, 159, 241)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>17</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="about_line">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>160</y>
      <width>811</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(97, 159, 241)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>17</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="predictor_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Display</family>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(71, 71, 71)</string>
    </property>
    <property name="text">
     <string>Predictor</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="about_button">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>110</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Display</family>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(71, 71, 71)</string>
    </property>
    <property name="text">
     <string>About</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="about" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>179</y>
      <width>1591</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>30</y>
       <width>272</width>
       <height>272</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/imgs/res/foto1000.png);
border-radius: 136px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Display</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(71, 71, 71)</string>
     </property>
     <property name="text">
      <string>Made by Juan David Pisco Jaimes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>132</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Display</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(71, 71, 71)</string>
     </property>
     <property name="text">
      <string>JuanDavidPiscoJaimes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>202</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Display</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(71, 71, 71)</string>
     </property>
     <property name="text">
      <string>juan-david-pisco-jaimes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/imgs/res/GitHub.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/imgs/res/linkedin.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>predictor</zorder>
   <zorder>about</zorder>
   <zorder>label</zorder>
   <zorder>main_line</zorder>
   <zorder>predictor_line</zorder>
   <zorder>about_line</zorder>
   <zorder>predictor_button</zorder>
   <zorder>about_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1607</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
